.github/workflows/modules-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         run: pip3 install 'avocado-framework' coverage
 
       - name: Run the ar module test
-        run: ./tests/test_module.py metadata/archive/ar.yml --coverage
+        run: ./tests/test_module.py metadata --coverage
```

Comment:
The title of this action should be changed now that all modules having metadata files will be tested.